client/my-sites/email/email-providers-stacked-comparison/index.tsx

```diff
@@ -2,6 +2,7 @@ import {
 	GOOGLE_WORKSPACE_BUSINESS_STARTER_YEARLY,
 	TITAN_MAIL_MONTHLY_SLUG,
 } from '@automattic/calypso-products';
+import { Domain } from '@automattic/data-stores/dist/types/site';
```

Comment:
Should this not be a type import? Also, I think we should not be "reaching in" to the package. If this is not exported at the top level of the module, we should consider adding it as an export.
```suggestion
import type { Domain } from '@automattic/data-stores';
```

Reply:
It seems, or I can't find a Domain type anywhere else... I'm using any at the moment